src/app/static/src/app/components/forms/DynamicFormControlGroup.vue

```diff
@@ -64,6 +71,10 @@
             required() {
                 return this.controlGroup.controls.length == 1
                     && this.controlGroup.controls[0].required
+            },
+            helpText() {
+                return this.controlGroup.controls.length == 1 ?
+                    this.controlGroup.controls[0].helpText : ""
```

Comment:
Doesn't this control show its own help text, in DynamicFormControl? Will this show it twice?

Reply:
Appreciate this is slightly confusing. A "control group" corresponds to a single row in the form, so if you want a control to appear in its own row, you create a control group with one control, and you give the control group a label, but not the control itself. The help text won't show up on the control because it only appears if the control has a label. 

Here for example, "Include ART data" is a group with 2 controls, whereas "Estimate neighbouring district ART attendance" and "Logit prior" are groups with 1 control each.

![Screenshot from 2019-12-04 17-00-02](https://user-images.githubusercontent.com/6736820/70163686-ccdbbb00-16b7-11ea-90db-ea26fa9424b5.png)

Arguably the help text should then be a property on the group rather than the control, but I'm a bit loathe to change the interface as would create more work for Rob.